
real	0m3.011s
user	0m0.323s
sys	0m0.521s
  File "server//file-input-output/python/output.txt", line 1
SyntaxError: Non-ASCII character '\xe4' in file server//file-input-output/python/output.txt on line 1, but no encoding declared; see http://python.org/dev/peps/pep-0263/ for details

real	0m0.036s
user	0m0.013s
sys	0m0.004s
